Templates/Ros2FleetRobotTemplate/README.md

```diff
@@ -0,0 +1,146 @@
+- [Proteus robot](https://github.com/o3de/o3de-extras/tree/development/Gems/ProteusRobot)
+- [WarehouseAssets](https://github.com/o3de/o3de-extras/tree/development/Gems/WarehouseAssets)
+
+Please make sure that `clang` was installed and configured. For details refer to [this section](https://www.o3de.org/docs/welcome-guide/requirements/#linux) of O3DE documentation.
+
+## Setup Instructions
+
+The following steps will assume the following:
+
+- You have ROS2 humble [installed](https://docs.ros.org/en/humble/Installation.html) and environment is [sourced](https://docs.ros.org/en/humble/Tutorials/Beginner-CLI-Tools/Configuring-ROS2-Environment.html#source-the-setup-files).
+- The O3DE gems, projects and templates will be placed in the O3DE home folder: `/home/${USER}/O3DE`.
+- You have the O3DE engine [downloaded and built](https://www.o3de.org/docs/welcome-guide/setup/setup-from-github) and it is located in `/home/${USER}/O3DE/Engines/Development`.
+
+In this tutorial, CLI tools will be used. It is also possible to use O3DE GUI to set up a project from template. See the [O3DE Project manager documentation](https://www.o3de.org/docs/user-guide/project-config/project-manager/) for more details. In such cases it is required to source your ROS2 distro before launching O3DE manager:
+
+```shell
+source /opt/ros/humble/setup.bash
+./build/linux/bin/profile/o3de
+```
+
+### 1. Install ROS2 packages
+
+```shell
+sudo apt install ros-${ROS_DISTRO}-ackermann-msgs ros-${ROS_DISTRO}-gazebo-msgs ros-${ROS_DISTRO}-control-toolbox
+```
+
+### 2. Download the template and asset gems
+
+For convenienience, we'll define a shell variables with O3DE folders:
+
+```shell
+export O3DE_HOME=/home/${USER}/O3DE
+export O3DE_ENGINE=${O3DE_HOME}/Engines/Development
+```
+
+Clone the `o3de-extras` repository containing the template and asset gems
+
+```shell
+mkdir -p ${O3DE_HOME}/Projects
+cd ${O3DE_HOME}/Projects
+git clone git@github.com:o3de/o3de-extras.git
+```
+
+Copy gems to the O3DE home.
+
+```shell
+mkdir -p ${O3DE_HOME}/Gems
+cp o3de-extras/Gems/ROS2 ${O3DE_HOME}/Gems
+cp o3de-extras/Gems/WarehouseAssets ${O3DE_HOME}/Gems
+cp o3de-extras/Gems/ProteusRobot ${O3DE_HOME}/Gems
+```
+
+Register these gems.
+
+```shell
+${O3DE_ENGINE}/scripts/o3de.sh register --gem-path ${O3DE_HOME}/Gems/ROS2
+${O3DE_ENGINE}/scripts/o3de.sh register --gem-path ${O3DE_HOME}/Gems/WarehouseAssets
+${O3DE_ENGINE}/scripts/o3de.sh register --gem-path ${O3DE_HOME}/Gems/ProteusRobot
+```
+
+### 3. Create a ROS2 project from the template
+
+Assign a name for the new project. In this example, it is assumed that it will be: `Ros2FleetRobotTest`, and it will be located in `$DEMO_BASE/Ros2FleetRobotTest` folder. 
+
+```shell
+export PROJECT_NAME=Ros2FleetRobotTest
+export PROJECT_PATH=${O3DE_HOME}/Projects/${PROJECT_NAME}
+${O3DE_ENGINE}/scripts/o3de.sh create-project --project-path $PROJECT_PATH --template-path ${O3DE_HOME}/Projects/o3de-extras/Templates/Ros2FleetRobotTemplate/ -f 
```

Comment:
#{O3DE_ENGINE} points to ${O3DE_HOME}/Engines/Development. There is no /scripts/o3de.sh there.

Reply:
Template was not registered.

Reply:
Removed.